Enterprise-CMCS/macpro-mdct-qmr
# OMS Rate Order Changes Global

### Description

Currently the OMS rates are displayed as Qualifiers with Categories underneath it. This new update changes the order of the text to have it match the way the information is displayed in Performance Measures. The Category is listed as the parent and then Qualifiers as the child of said category.

Before
<img width="1230" alt="Screenshot 2023-06-02 at 11 40 54 AM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/127151429/4a3f3215-7db4-416d-a05b-9e4fcf22c4fd">

After
<img width="1184" alt="Screenshot 2023-06-02 at 11 41 21 AM" src="https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/127151429/b1d56812-4abb-4fc7-b27a-8fb13fde4cc0">


### Related ticket(s)


MDCT-2444

---
### How to test

1) Login QMR, any account
2) Select Reporting Year 2023
3) Go into Adult Core Measures
4) Select any of these measures: [ AMM-AD ][ CCP-AD ][ FUA-AD ][ FUH-AD ][ FUM-AD ][ HBD-AD ][ IET-AD ]
5) To get OMS to display, select the first radio button for the first 3 questions. This will enable the Performance Measure section.
6) Scroll down to Performance Measure, fill in the N/D/R for all of them. 
      Note: If the N/D/R for a certain qualifier is not filled, the qualifier will not show up as an option in the OMS selection.
7) Scroll down to OMS and select the classifications until you can see the category & qualifier selections. 
8) Compare order with the Performance Measure N/D/R. They should be structured the same
9) Test validation to make sure data is saving / validating correctly.

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
